integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/Hooks.java

```diff
@@ -36,20 +40,28 @@ public void before(Scenario scenario) {
 
     @After
     public void cleanUp() {
-        String token = BridgeUtils.retrieveBridgeToken();
-        String bridgeId = context.getBridgeId();
-        if (BridgeResource.getBridgeList(token).getItems().stream()
-                .anyMatch(b -> b.getId().equals(context.getBridgeId()))) {
-            BridgeResponse bridge = BridgeResource.getBridgeDetails(context.getManagerToken(), bridgeId);
-            if (bridge.getStatus() == BridgeStatus.READY) {
-                if (ProcessorResource.getProcessorList(token, bridgeId).getSize() > 0) {
-                    ProcessorResource.getProcessorList(token, bridgeId).getItems().stream()
-                            .forEach(p -> ProcessorResource.deleteProcessor(token, bridgeId, p.getId()));
-                    Awaitility.await().atMost(Duration.ofMinutes(2)).pollInterval(Duration.ofSeconds(5))
-                            .until(() -> ProcessorResource.getProcessorList(token, bridgeId).getSize() == 0);
-                }
-            }
-            BridgeResource.deleteBridge(token, bridgeId);
-        }
+        System.out.println("Started cleanup");
+        // Remove all bridges/processors created
+        context.getAllBridgeNames().stream().map(context::getBridge).map(BridgeContext::getBridgeId)
```

Comment:
May be simpler to retrieve all Bridge instances from context instead? Then you don't need to remap to Bridge.

Reply:
updated